Lib/unittest/case.py

```diff
@@ -463,15 +483,8 @@ def defaultTestResult(self):
         return result.TestResult()
 
     def shortDescription(self):
-        """Returns a one-line description of the test, or None if no
-        description has been provided.
-
-        The default implementation of this method returns the first line of
-        the specified test method's docstring.
-        """
-        doc = self._testMethodDoc
-        return doc and doc.split("\n")[0].strip() or None
-
+        """ Return a one-line description of the test, if any; otherwise None. """
```

Comment:
Not sure that the previous description is entirely relevant. Also does saying "The default implementation" add any useful information? How about:

```
"""Returns a one-line description of the test, or None if no
description has been provided.

This method returns the summary line of
the specified test method's docstring, as per PEP-257.
"""
```